### ISSUE TITLE
Menu rotating arrows not displayed

### ISSUE THREAD

**Issue by doolio** [cncf/dot-org-hugo-theme#41]
The demo site has the following:

![image](https://github.com/cncf/dot-org-hugo-theme/assets/13859009/3c1b60a5-b73f-4b53-a115-8302e7937904)

However, on a site I'm building I don't see these elements.

![image](https://github.com/cncf/dot-org-hugo-theme/assets/13859009/9ef1a370-fd3e-4b12-a480-63ec997e9053)

The only SCSS I override is the `styles.scss` file to introduce a `_news.scss` file which is based on the `_blog.scss` theme file.

I do have a `home.scss` file based on that from the Deep Casuality site but I don't think that is the issue because these arrows are visible on that site.

I presume these rotating arrows are driven by the `_header.scss` file but I have not overridden this file. Thanks for your time.

**Comment by thetwopct** [comment]
Hard to give feedback without seeing the code tbh.

The arrows are attached to the class ".menu-item-has-children" - which is applied if the top menu item has children 

https://github.com/cncf/dot-org-hugo-theme/blob/47e7ade287b2541ee3b6702604297c8c98677472/layouts/partials/header.html#L22

https://github.com/cncf/dot-org-hugo-theme/blob/47e7ade287b2541ee3b6702604297c8c98677472/assets/scss/_header.scss#L249

**Comment by doolio** [comment]
Thank you. I think my problem is I have my menus defined in a `menus.yaml` file (under a `main:` key) in the `config/_default/` directory rather than under a `menu:` key in `config/_default/languages.yaml`.

**Comment by thetwopct** [comment]
I see. The "menu:" in languages.yaml and the "menus" config file may be conflicting. Remove the menu: and its children from the language file. The existing code at L18 should still work if your file is in the _default config folder and called menus.yaml.

https://github.com/cncf/dot-org-hugo-theme/blob/47e7ade287b2541ee3b6702604297c8c98677472/layouts/partials/header.html#L18

Oh and menus.yaml will need to be properly structured and indented, Hugo can be a bit tricky with that

**Comment by doolio** [comment]
Sorry I should have been more clear. I removed the `menu:` key and its children from the `languages.yaml` file and put its children starting from the `main:` key into a `menus.yaml` file (i.e. this file does not contain a `menu:` key). This I believe is in line with splitting one's config into separate files. 

My site is not multi-lingual so I suppose I could remove the `languages.yaml` file entirely. I did remove the `en:` key from this file thinking that may resolve it as the Hugo docs describe menu files such as `menus.en.yaml`, `menus.es.yaml` etc. when a site is multilingual.
